README.md

```diff
@@ -83,6 +85,17 @@ Note that the external-provisioner does not scale with more replicas. Only one e
 
 * `--leader-election-type`: This option was used to choose which leader election resource type to use. Currently, the option defaults to `endpoints`, but will be removed in the future to only support `Lease` based leader election.
 
+### Topology support
+When `Topology` feature is enabled and the driver specifies `VOLUME_ACCESSIBILITY_CONSTRAINTS` in its plugin capabilities, external-provisioner prepares `CreateVolumeRequest.AccessibilityRequirements` while calling `Controller.CreateVolume`. The driver has to consider these topology constraints while creating the volume. Below table shows how these `AccessibilityRequirements` are prepared:
+
+Delayed binding | Strict Topology | AllowedTopologies | AccessabilityRequirements
+:---: |:---:|:---:|:---|
+Yes | Yes | Yes/No | `Requisite` = `Preferred` = Selected node topology
```

Comment:
What is "AllowedTopologies"? To be on the safe side and ensure that all readers will understand the table, better add a short explanation or link for each column.

The spelling of the column headers is inconsistent (First word upcase, title case, camel case). Not sure which one is best, but better pick just one.

"Yes/No" confused me at first. Perhaps replace it with "ignored", "irrelevant" or "*" (wildcard)? When using "*", a short comment under the table would be useful.

Other than that the table is very useful, thanks.

Reply:
> What is "AllowedTopologies"? To be on the safe side and ensure that all readers will understand the table, better add a short explanation or link for each column.
> 
Added links to column headers.
> The spelling of the column headers is inconsistent (First word upcase, title case, camel case). Not sure which one is best, but better pick just one.
> 
Fixed to 'First word upper case'.

> "Yes/No" confused me at first. Perhaps replace it with "ignored", "irrelevant" or "_" (wildcard)? When using "_", a short comment under the table would be useful.
> 
Used "Irrelevant" in place of "Yes/No".